src/main/java/uk/gov/companieshouse/officerfiling/api/validation/OfficerUpdateValidator.java

```diff
@@ -175,6 +190,15 @@ public boolean doesNationalityMatchChipsData(OfficerFilingDto dto, AppointmentFu
         return chipsNationalities.length < 3 || matchesChipsField(dto.getNationality3(), chipsNationalities[2]);
     }
 
+    public boolean doesOccupationMatchChipsData(OfficerFilingDto dto, AppointmentFullRecordAPI appointmentFullRecordAPI) {
+        if (appointmentFullRecordAPI.getOccupation() == null ||
+                appointmentFullRecordAPI.getOccupation().equalsIgnoreCase("NONE") && !dto.getOccupation().equalsIgnoreCase("NONE")) {
```

Comment:
dto.getOccuption() could be "" or null and that means same as "NONE" so:
... && (!dto.getOccupation().equalsIgnoreCase("NONE") || dto.isEmpty())